common-api/src/main/java/org/pmiops/workbench/cdr/CdrDbConfig.java

```diff
@@ -93,15 +93,16 @@ public CdrDataSource(CdrVersionDao cdrVersionDao,
           // explicitly initialize the pool parameters here. We override the primary connection
           // info, as the autowired PoolConfiguration is initialized from the same set of properties
           // as the workbench DB.
-          basePoolConfig.setUsername(dbUser);
```

Comment:
So I think you need to copy the configuration from basePoolConfig somehow... @calbach  to confirm?

Reply:
this code worked fine locally with 2 cdr versions.

Reply:
Yikes - this is a bad bug - my fault most likely. Have to remember that Beans are singleton by default... Thanks for debugging and catching.

I think this is fine, as long as we're copying all of the properties from that pool config.